### PR TITLE
Update doc links to docs.redhat

### PR DESCRIPTION
**Pull request by aduquett**
https://issues.redhat.com/browse/RHOAIENG-13596

## Description
Documentation links have changed from access.redhat.com and docs.openshift.com to docs.redhat.com.

## How Has This Been Tested?
Confirmed that the new links are valid.

### CODE REVIEW

File: manifests/rhoai/addon/apps/nvidia/nvidia-app.yaml [hunk 1]
```diff
@@ -19,15 +19,14 @@ spec:
   csvName: gpu-operator-certified
   docsLink: https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/contents.html
   quickStart: gpu-quickstart
-  getStartedLink: https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html/managing_users_and_user_resources/enabling-gpu-support-in-openshift-data-science_user-mgmt
+  getStartedLink: https://docs.redhat.com/en/documentation/red_hat_openshift_ai_cloud_service/1/html/managing_resources/managing-cluster-resources_cluster-mgmt#enabling-nvidia-gpus_cluster-mgmt
```
Comment (MelissaFlinn) [inline manifests/rhoai/addon/apps/nvidia/nvidia-app.yaml:22]: managing_resources/managing-cluster-resources_cluster-mgmt#enabling-nvidia-gpus_cluster-mgmt
-> 
managing_rhoai/enabling_accelerators#enabling-nvidia-gpus_managing-rhoai)

File: manifests/rhoai/addon/apps/nvidia/nvidia-app.yaml [hunk 2]
```diff
@@ -36,4 +35,4 @@ spec:
     The NVIDIA GPU Operator uses the operator framework within Kubernetes to automate the management of all NVIDIA software components needed to provision GPU. These components include the NVIDIA drivers (to enable CUDA), Kubernetes device plugin for GPUs, the NVIDIA Container Toolkit, automatic node labelling using GFD, DCGM based monitoring and others.
 
     Documentation :
-    [Enabling GPU Support in OpenShift AI](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html/managing_users_and_user_resources/enabling-gpu-support-in-openshift-data-science_user-mgmt)
+    [Enabling GPU Support in OpenShift AI](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_cloud_service/1/html/managing_openshift_ai/enabling_accelerators#enabling-nvidia-gpus_managing-rhoai)
```
Comment (MelissaFlinn) [inline manifests/rhoai/addon/apps/nvidia/nvidia-app.yaml:38]: managing_openshift_ai
->
managing_rhoai

**Comment by aduquett** [comment]
Note that these links will not be live until 2.14 docs are published:
https://docs.redhat.com/en/documentation/red_hat_openshift_ai_cloud_service/1/html/managing_openshift_ai/enabling_accelerators#enabling-nvidia-gpus_managing-rhoai
https://docs.redhat.com/en/documentation/red_hat_openshift_ai_cloud_service/1/html/managing_openshift_ai/managing-users-and-groups
https://docs.redhat.com/en/documentation/red_hat_openshift_ai_cloud_service/1/html/managing_openshift_ai/managing-users-and-groups#adding-users-to-user-groups_managing-rhoai

**Review by Gkrumbach07 — APPROVED** [review]
/lgtm

the remaining url changes will all take effect in 2.15 so this PR is good to go.

**Pull request merged**